### PR TITLE
[Docs] Tweak example excel file to prevent csvkit crash during tutorial

### PR DESCRIPTION
**Pull request by johnhkchen**
**tl;dr:** I re-opened the tutorial file in excel and now `in2csv` likes it again.

Hello,

I was working through the tutorial problems and got stuck on [step 1.3](https://csvkit.readthedocs.io/en/latest/tutorial/1_getting_started.html#getting-the-data).

The file grabbed by `curl -L -O https://raw.githubusercontent.com/wireservice/csvkit/master/examples/realdata/ne_1033_data.xlsx`, for whatever reason, generates an error when I ran `in2csv ne_1033_data.xlsx` (as per the instructions).

```
ValueError: Worksheet is unsized, use calculate_dimension(forWorksheet is unsized, use calculate_dimensione=True)
```

I think the error is generated by openpyxl, but don't have the expertise to delve deeper. Luckily, opening and saving the excel file using excel seemed to have cleared up the issues, so I'm creating this PR to hopefully clear up a discrepancy between the tutorial instructions and actual results.

in2csv 1.3.0
Ubuntu + Python 3.11/12

Steps to reproduce:

- Follow instructions starting from https://csvkit.readthedocs.io/en/latest/tutorial/1_getting_started.html
- Download the file as instructed, run `in2csv ne_1033_data.xlsx`
- Get ValueError
- Open file in Excel, save
- Run same command, output as expected

### CODE REVIEW

**Comment by jpmckinney** [comment]
Hi, this is actually a bug in agate-excel. I've released agate-excel 0.4.1. You can upgrade with:

```
pip install -U agate-excel==0.4.1
```
